VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/joh-waiting-room/joh-waiting-room.component.html

```diff
@@ -106,21 +108,21 @@ <h1 class="govuk-heading-xl govuk-!-padding-top-5">Your video hearing {{ getConf
         <div *ngIf="currentTime" class="govuk-panel" [ngClass]="getCurrentTimeClass()">
             <div class="govuk-body govuk-!-margin-bottom-0" [ngClass]="getCurrentTimeClass()">
                 <div *ngIf="hearing.isOnTime() || hearing.isStarting() || hearing.isDelayed()">
-                    <h2 class="govuk-heading-m white">Please wait</h2>
-                    <p class="white">Your video hearing is scheduled to begin at {{ hearing.scheduledStartTime | date: 'HH:mm' }}</p>
+                    <h2 class="govuk-heading-m white">{{'joh-waiting-room.please-wait' | translate}}</h2>
+                    <p class="white">{{'joh-waiting-room.hearing-starting-at' | translate}} {{ hearing.scheduledStartTime | date: 'HH:mm' }}</p>
                 </div>
 
                 <div *ngIf="hearing.isClosed()">
-                    <h2 class="govuk-heading-m white">This hearing has finished. You may now sign out.</h2>
+                    <h2 class="govuk-heading-m white">{{'joh-waiting-room.hearing-closed' | translate}}</h2>
                 </div>
 
                 <div *ngIf="hearing.isPaused()" class="hearing-title-mt--20">
-                    <p class="white">The judge will restart the hearing when they are ready. Please stay near your screen.</p>
+                    <p class="white">{{'joh-waiting-room.hearing-paused' | translate}}</p>
                 </div>
 
                 <div *ngIf="hearing.isSuspended()">
-                    <h2 class="govuk-heading-m">We're having technical issues</h2>
-                    <p>All participants are affected. Stay near your screen for further updates. There's no need to call us.</p>
+                    <h2 class="govuk-heading-m">{{'joh-waiting-room.hearing-suspended-technical-message-1' | translate}}</h2>
```

Comment:
`hearing-suspended-technical-title`

Reply:
and below could be just `hearing-suspended-message`

Reply:
Done